packager/media/replicator/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Google Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+add_library(media_replicator STATIC
+        replicator.cc)
```

Comment:
style nit: align with previous line arguments

Reply:
Actually, no, I was wrong.  This should be 4-space indentation.  I was misremembering the convention we were using in CMakeLists.